.env.local.example

```
@@ -4,6 +4,13 @@ SELF_HOSTED=true
 # Enable Twelve market data (careful, this will use your API credits)
 TWELVE_DATA_API_KEY=
 
+# OpenID Connect for development
+OIDC_ISSUER=
+OIDC_CLIENT_ID=
+OIDC_CLIENT_SECRET=
+OIDC_REDIRECT_URI=http://localhost:3000/auth/openid_connect/callback
+TWELVE_DATA_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The TWELVE_DATA_API_KEY key is duplicated

(DuplicatedKey)